App/App_iOS.swift

```diff
@@ -1,10 +1,4 @@
-//
```

Comment:
Copyright is not owned by Kiwix. Headers should be changed in a dedicated PR, see ticket #575. Lets keep this PR for the custom apps.